packages/service/src/domain/commands/processors/tag-add-command.ts

```diff
@@ -0,0 +1,30 @@
+import Command from '../model/command';
+import CommandProcessingResponse, { makeSingleTextProcessingResponse } from '../model/command-processing-response';
+import CommandDefinition from '../model/command-definition';
+import XSoundsService from '../../x-sounds/x-sounds-service';
+
+async function tagAddCommandProcessor(command: Command): Promise<CommandProcessingResponse> {
+  const commandArgs = command.getArgsByDelimiter('|');
+
+  if (commandArgs.length < 2) {
+    throw new Error('Zbyt mała liczba argumentów');
+  }
+
+  const [tagName, soundName] = commandArgs;
+
+  await XSoundsService.instance.addTag(soundName, tagName);
+  return makeSingleTextProcessingResponse(`Dodano tag "${tagName}" do dźwięku ${soundName}`, false);
+}
+
+const tagAddCommandDefinition: CommandDefinition = {
```

Comment:
`tag` seems like complicated functionality
What do you think about adding `tag-help` which displays help for all `tag-xyz` commands?

Reply:
we could even provide `tag` as shortKey for it

Reply:
So like `/fm help` but with `tag-*` filter? Or some other information that is not included in `/fm help`?

Reply:
like help with tag filter 👍🏻

Reply:
I'm not sure about that. Adding new command that does basically what `/fm help` seems unnecessary 🤔 .
Maybe we can test how `/fm help` looks with new commands on Slack and then decide?